SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularityMailer.java

```diff
@@ -477,6 +497,10 @@ public void sendRequestScaledMail(SingularityRequest request, Optional<Singulari
   }
 
   public void sendRequestRemovedMail(SingularityRequest request, Optional<String> user, Optional<String> message) {
+    if (!maybeSmtpConfiguration.isPresent()) {
```

Comment:
Actually, I was going to remove the check in sendRequestMail and leave those in.
The justification being that I'd rather have it be standard that the public methods check and then can safely call the private methods than split between the two.